docs/buildah.md

```diff
@@ -46,23 +46,23 @@ be used, as the default behavior of using the system-wide configuration
 
 **--root** **value**
 
-Storage root dir (default: "/var/lib/containers/storage")
+Storage root dir (default: "/var/lib/containers/storage" for UID 0, "$HOME/containers/storage" for other users)
```

Comment:
this should be updated to reflect the last changes, `$HOME/containers/storage` is not used anymore

Reply:
Whoops, fixing.